src/components/orders/StatusLabel/index.tsx

```diff
@@ -55,15 +104,15 @@ type ShimmingProps = {
   shimming?: boolean
 }
 
-const Label = styled.div<DisplayProps & ShimmingProps>`
+const Label = styled.div<DisplayProps & ShimmingProps & PartiallyTagProps>`
   font-weight: ${({ theme }): string => theme.fontBold};
-  text-transform: capitalize;
```

Comment:
@alongoni 
I have not been able to get the formatting to work for text with: 
```css
&::first-letter {
    text-transformation: uppercase;
}
```

So I've had to go old school with JavaScript [L182](https://github.com/cowprotocol/explorer/blob/155-fix-row-surplus/src/components/orders/StatusLabel/index.tsx#L182)